src/lib/openapi/spec/instance-admin-stats-schema.ts

```diff
@@ -4,57 +4,123 @@ export const instanceAdminStatsSchema = {
     $id: '#/components/schemas/instanceAdminStatsSchema',
     type: 'object',
     additionalProperties: false,
+    description:
+        'Information about an instance and statistics about usage of various features of Unleash',
     required: ['instanceId'],
     properties: {
         instanceId: {
             type: 'string',
+            description:
+                'A unique identifier for this instance. Generated by the database migration scripts at first run. Typically a UUID.',
+            example: 'ed3861ae-78f9-4e8c-8e57-b57efc15f82b',
         },
         timestamp: {
             type: 'string',
             format: 'date-time',
             nullable: true,
+            description: 'When these statistics were produced',
+            example: '2023-06-12T12:25:06Z',
         },
         versionOSS: {
             type: 'string',
+            description:
+                'The version of Unleash OSS that is bundled in this instance',
+            example: '5.1.7',
         },
         versionEnterprise: {
             type: 'string',
+            description:
+                'The version of Unleash Enterprise that is bundled in this instance',
+            example: '5.1.7',
         },
         users: {
             type: 'number',
+            description: 'The number of users this instance has',
+            example: 8,
         },
         featureToggles: {
             type: 'number',
+            description: 'The number of feature-toggles this instance has',
+            example: 47,
         },
         projects: {
             type: 'number',
+            description: 'The number of projects defined in this instance.',
+            example: 3,
         },
         contextFields: {
             type: 'number',
+            description:
+                'The number of context fields defined in this instance.',
+            example: 7,
         },
         roles: {
             type: 'number',
+            description: 'The number of roles defined in this instance',
+            example: 5,
         },
         groups: {
             type: 'number',
+            description: 'The number of groups defined in this instance',
+            example: 12,
         },
         environments: {
             type: 'number',
+            description: 'The number of environments defined in this instance',
+            example: 3,
         },
         segments: {
             type: 'number',
+            description: 'The number of segments defined in this instance',
+            example: 19,
         },
         strategies: {
             type: 'number',
+            description: 'The number of strategies defined in this instance',
+            example: 8,
```

Comment:
We **could** mark all these numbers in the admin stats schema with a `minimum: 0`. That would be more accurate. However, because this is a response schema, we're not in any danger of the users submitting wrong data based on this, so it's not that important. 

What do you think? I think it'd be nice, but I'll leave it up to you.

Reply:
Sure, I mean, when parsing this that would allow me to use unsigned types safely. Compared to how little work it is to add, happy to add it.